src/components/Checkbox/index.tsx

```diff
@@ -61,19 +61,17 @@ export class Checkbox extends React.Component<CheckboxRadioProps> {
     });
   }
 
-  private get textStyle(): StyleProp<TextStyle> {
+  private get textStyle(): TextStyle {
     const { disabled } = this.props;
 
-    let finalStyle: any = {
+    return {
       color: getColor(disabled ? 'textDisabled' : 'textPrimary'),
       marginLeft: 8,
       lineHeight: 20,
     };
-
-    return StyleSheet.create(finalStyle);
```

Comment:
This is actually an invalid thing ... it expects sort of className → style mapping